virttest/utils_test/qemu.py

```diff
@@ -140,23 +155,21 @@ def setup_win_driver_verifier(session, driver, vm, timeout=300):
     :param driver: The driver which needs enable the verifier.
     :param vm: VM object.
     :param timeout: Timeout in seconds.
-    :return: Session after reboot.
     """
-    verifier_setup_cmd = "verifier.exe /standard /driver %s.sys" % driver
-    if driver:
+    verifier_status = check_driver_verifier(session, driver)[1]
+    if not verifier_status:
+        error_context.context("Enable %s driver verifier" % driver)
```

Comment:
hi, if didn't specify the logging function ,will this context be displayed in debug.log,thanks.